index.html

```diff
@@ -5,9 +5,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="Cross-Origin-Opener-Policy" content="same-origin-allow-popups" />
     <title>ROAR</title>
+    <link rel="icon" href="public/favicon.ico" sizes="48x48" />
+    <link rel="icon" href="public/favicon.svg" sizes="any" type="image/svg+xml" />
+    <link rel="apple-touch-icon" href="/apple-touch-icon-180x180.png" />
   </head>
   <!-- Google tag (gtag.js) -->
   <script async src="https://www.googletagmanager.com/gtag/js?id=G-YYE3YN0S99"></script>
+  <script>
+    if ('serviceWorker' in navigator) {
```

Comment:
You may want to move this to a separate file and call it during the initialization of the app. This file in the future can do other things. Please take a look at an example of how this file usually looks here:
https://github.com/cra-template/pwa/blob/main/packages/cra-template-pwa-typescript/template/src/serviceWorkerRegistration.ts